### PR TITLE
Smiles processing refactored

### PR DESCRIPTION
**Pull request by 243083df**

### CODE REVIEW

File: modules/chat/sc2tv.py [hunk 1]
```diff
@@ -84,14 +77,12 @@ def __init__(self, user, text, subscr):
...
     def process_smiles(self, smiles):
-        smiles_array = re.findall(SMILE_REGEXP, self._text)
-        for smile in smiles_array:
-            for smile_find in smiles:
-                if smile_find['code'] == smile.lower():
-                    if allow_smile(smile_find, self._subscriptions):
-                        self._text = self._text.replace(SMILE_FORMAT.format(smile),
-                                                        EMOTE_FORMAT.format(smile))
-                        self._emotes.append(Emote(smile, smile_find['url']))
+        finded = re.findall(SMILE_REGEXP, self._text)
```
Comment (DeForce) [inline modules/chat/sc2tv.py]: finded - there is no such word. 

rename to found_smiles

**Comment by 243083df** [comment]
Hello?

**Comment by DeForce** [comment]
Will merge after #413

**Comment by DeForce** [comment]
Rebase on latest develop with 1 commit and I will merge

**Pull request closed without merging**